infrastructure/scripts/execution-plan/plan-pr.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 set -ueo pipefail
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
```

Comment:
I would write this as:
```bash
script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
```

Since you know you use bash, you do not need the `:-$0` fallback. Also the output redirects seem a bit much. And the `--` as first argument was surprising to me -- it works without and the `--` argument is not documented in `bash_builtins(1)` for my version of bash (5.1.8).